README.md

```diff
@@ -1,24 +1,159 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<div align="center">
 
-Things you may want to cover:
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
 
-* Ruby version
+  <h3>Microverse</h3>
 
-* System dependencies
+</div>
 
-* Configuration
+# 📗 Table of Contents
 
-* Database creation
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
 
-* Database initialization
 
-* How to run the test suite
+# 📖 [Savier] <a name="about-project"></a>
 
-* Services (job queues, cache servers, search engines, etc.)
+> Savier is a rails application that allows user keep track of their spending
 
-* Deployment instructions
 
-* ...
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://getbootstrap.com/">Bootstrap</a></li>
+       <li><a href="https://yarnpkg.com/">Yarn</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+   <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+          <li><a href="https://nodejs.org/en/">node</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+   <li><a href="https://www.postgresql.org/">Postgresql</a></li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+- **[User can add transaction]**
+- **[User can delete transaction]**
+- **[User can add and delete category]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
```

Comment:
- [ ] [REQUIRED] Kindly ensure that your deployment is successful. You have deployed your project but however, but you also need to deploy the database for the project to be ultimately deployed. Below is the error displayed whenever I try to access your deployment link.
<img width="873" alt="Screenshot 2022-11-24 at 19 52 53" src="https://user-images.githubusercontent.com/58692590/203851985-1274e7e6-f244-44c5-8055-e8c4e655ca57.png">